source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRImageRenderer.mm

```diff
@@ -198,11 +198,14 @@ - (void)configUpdateForUIImageView:(ACRView *)rootView acoElem:(ACOBaseCardEleme
     constraints[0].priority = priority;
     constraints[1].priority = priority;
 
+    // keeping all precisions are not necessary, and 744W X 84H and its multiples cause a crash.
+    // rounding off to 100th points.
+    // To make the constraints work, the rounded value has to become 1 when the two ratios are multiplied
+    const CGFloat precision = 100;
```

Comment:
do we need to do something similar in these places? 

https://github.com/microsoft/AdaptiveCards/blob/0e505ff09c9b6e5c46da5a23c2848b3ef7a108c0/source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm#L193-L199

https://github.com/microsoft/AdaptiveCards/blob/0e505ff09c9b6e5c46da5a23c2848b3ef7a108c0/source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRMediaRenderer.mm#L201-L207

https://github.com/microsoft/AdaptiveCards/blob/0e505ff09c9b6e5c46da5a23c2848b3ef7a108c0/source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRMediaTarget.mm#L172-L179

if so, we should probably make a helper...

Reply:
I don't it will hurt. let me make the change.